### ISSUE TITLE
fastlane match migrate leaves user stuck at picking role for service account

### ISSUE THREAD

**Issue by tpaik18** [fastlane/fastlane#15296]
Reopening https://github.com/fastlane/fastlane/issues/13971 reported by @janpio 

Using fastlane 2.130.0.

`match migrate` tells me to:

```
[21:16:30]: Now create a new JSON auth file by clicking on
[21:16:30]:
[21:16:30]:              1. Create credentials
[21:16:30]:              2. Service account key
[21:16:30]:              3. App Engine default service account
...
```
But "App Engine default service account" is not an option, only "New service account".
There I have a "Role" dropdown, but nothing that fully matches "App Engine default service account". I tried with the "App Engine" entries there, but then I get this error later after I have downloaded and copied the JSON to gc_keys.json:
```
/Library/Ruby/Gems/2.3.0/gems/google-cloud-storage-1.16.0/lib/google/cloud/storage/service.rb:570:in 
`rescue in execute': [!] forbidden: xxxxxx-signing@xxxxxxxx.iam.gserviceaccount.com 
does not have storage.buckets.list access to project xxxxxxxxxxx. 
(Google::Cloud::PermissionDeniedError)
```

Can you please update the instructions as to which Role in that big nested dropdown we should pick to have the proper access level?

**Comment by rickclephas** [comment]
The documentation is referring to the already existing "App Engine default service account" service account.
The service accounts email ends with `@appspot.gserviceaccount.com`.
Under "Actions" (the three dots on the right) there is an option "Create key".

Using that key should work as this service account has the "Editor" role.

@janpio IMHO the documentation shouldn't encourage users to use this service account.
This service account has **[1945](https://console.cloud.google.com/iam-admin/roles/details/roles<editor)** permissions!
From my understanding `match` only needs access to a Google Cloud Storage bucket so if you ask me we should have documentation on creating a service account with the required permissions only.

**Comment by rickclephas** [comment]
Alright it seems newly created projects don't have the App Engine default service account.
The service account is created after you [create](https://console.cloud.google.com/appengine/start) an App Engine application.

Creating a new service account with the "Editor" role has the same effect.

**Comment by janpio** [comment]
Thanks @rickclephas, I unfortunately could these instructions to work because of 

> Alright it seems newly created projects don't have the App Engine default service account.
> The service account is created after you [create](https://console.cloud.google.com/appengine/start) an App Engine application.

... and I had created multiple issues about this in the past.

Can you summarize your suggestions or create a Pull Request to the code and docs?
